ci/build-appimage.sh

```diff
@@ -45,6 +45,8 @@ export PIP_WORKDIR="$REPO_ROOT"
 export OUTPUT=appimagelint-x86_64.AppImage
 export VERSION="$SETUPPY_VERSION-git$COMMIT"
 
+mkdir -p AppDir/usr/share/metainfo/ && cp $REPO_ROOT/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```

Comment:
```suggestion
mkdir -p AppDir/usr/share/metainfo/ && cp "$REPO_ROOT"/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```

Unsichere Evaluation. Variablen bitte immer in " wrappen (so "eng" wie möglich).